packages/loader/src/TextureLoader.ts

```
@@ -0,0 +1,204 @@
+            }
+          }
+        })
+        .catch(reject);
+    });
+  }
+
+  private _loadHDR(
+    buffer: Uint8Array,
+    item: LoadItem,
+    resourceManager: ResourceManager
+  ): AssetPromise<Texture2D> {
```

ESLint

[error] Replace `⏎····buffer:·Uint8Array,⏎····item:·LoadItem,⏎····resourceManager:·ResourceManager⏎··` with `buffer:·Uint8Array,·item:·LoadItem,·resourceManager:·ResourceManager`

(prettier/prettier)